package.json

```diff
@@ -98,20 +90,18 @@
     "react-ace": "9.4.0",
     "react-autosize-textarea": "7.1.0",
     "react-dom": "^17.0.1",
-    "resize-observer-polyfill": "1.5.1",
     "sass": "1.49.8",
     "sass-loader": "9.0.3",
     "style-loader": "1.2.1",
     "tailwindcss": "2.0.2",
     "typescript": "4.5.5",
-    "uuid": "8.3.2",
```

Comment:
I added too many NPM packages. This one is replaced with a 1-liner of vanilla JavaScript 🤣